File: Extra/inheritance.cpp
#include<iostream>
using namespace std;

class Employee{
    private:
    int val;
    string name;

    Employee(int val, string name){
        this->val = val;
        this->name = name;
    }

    public: string PrintName(){
        return this->name;
    }
};

int main()
{
    // Employee emp1 = Employee(1, "Gourav kumar singh");
    emp1.PrintName();
}

File: Extra/inheritance.py
class Employee():
    def __init__(self, val, name):
        self.val = val
        self.name = name
    
    def PrintName(self):
        return self.name

class Freshers(Employee):
    def __init__(self, age):
        self.age = 12
        
class class2(Employee):
    pass


if __name__ == "__main__":
    emp = class2()
    